packages/synchronizer/src/handlers/storageHandler.ts

```diff
@@ -0,0 +1,76 @@
+import {Document, parse} from 'yaml';
+import {mkdirp} from 'mkdirp';
+import {existsSync, readFileSync} from 'fs';
+import {readFile, writeFile} from 'fs/promises';
+import {dirname, join, normalize} from 'path';
+
+export abstract class StorageHandler<DATA_FORMAT> {
```

Comment:
Bit nitpick. For generics I think a Java styled approach is cleaner? `TData` instead of `DATA_FORMAT`? I had to double check because I thought this was a constant but that made no sense for a type.

Reply:
Sure 👍 I'm used to another Java conventions which is just using `T`, `Y`, `Z`, which is not really readable IMHO. So anything more readable makes sense, and this one looks good 👍